components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/AwesomeBarDefaults.kt

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.compose.browser.awesomebar
+
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+/**
+ * Contains the default values used by the AwesomeBar.
+ */
+object AwesomeBarDefaults {
+    /**
+     * Creates an [AwesomeBarColors] that represents the default colors used in an AwesomeBar.
+     *
+     * @param background The background of the AwesomeBar.
+     * @param title The text color for the title of a suggestion.
+     * @param description The text color for the description of a suggestion.
+     */
+    @Composable
+    fun colors(
+        background: Color = MaterialTheme.colors.background,
```

Comment:
if these are the default colors, should this composable function not take any parameters and instead just pass in the defaults directly to the "AwesomeBarColors" constructor?

Reply:
That's what I wanted to do initially. But accessing the `MaterialTheme` properties requires to be in a `Composable` scope. So I moved this to this function, which an app can call to create a customized color scheme. I took this idea form the "components style" section in the docs:
https://developer.android.com/jetpack/compose/themes#component-styles